code/modules/paperwork/ticketmachine.dm

```diff
@@ -167,9 +167,13 @@
 		user.adjust_fire_stacks(1)
 		user.IgniteMob()
 
+
+/obj/machinery/ticket_machine/attack_ai(mob/user)//If the user is the AI
+	return //do nothing
+
 /obj/item/ticket_machine_ticket
 	name = "Ticket"
-	desc = "A ticket which shows your place in the Head of Personnel's line. Made from Nanotrasen patented NanoPaper�. Though solid, its form seems to shimmer slightly. Feels (and burns) just like the real thing."
+	desc = "A ticket which shows your place in the Head of Personnel's line. Made from Nanotrasen patented NanoPaper�. Though solid, its form seems to shimmer slightly. Feels (and burns) just like the real thing."
```

Comment:
Also revert this change please. Looks like you are using a different character set and thus saving it corrupted the character there